src/js/eventInTimeAware.js

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // A grade that deals in time,
+    // and things related to time
+    fluid.defaults("floe.dashboard.eventInTimeAware", {
+        gradeNames: "fluid.modelComponent",
+        model: {
+            // A series of key-timestamp pairs
+            timeEvents: {
+                // created: "",
+                // lastModified: ""
+            },
+            formattedDates: {
+                "created": null,
```

Comment:
`null` is an inappropriate initial value for model material, which will likely confuse users and other relays. The fact that the component deals in particular model fields which have particular values/types is best expressed in comments. One day, of course, we will have model schema grades widespread.